cln-rpc/src/primitives.rs

```diff
@@ -260,6 +260,9 @@ impl ShortChannelId {
     pub fn outnum(&self) -> u16 {
         self.0 as u16 & 0xFFFF
     }
+    pub fn to_u64(&self) -> u64 {
+        self.0
```

Comment:
why not implement `Into<u64>` and work with the rust compiler?

Reply:
Maybe add an impl that uses to_u64 and we're golden 🤗